modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -316,6 +316,9 @@
 
 (defmethod driver/supports? [:bigquery-cloud-sdk :foreign-keys] [_ _] true)
 
+(defmethod driver/database-supports? [:bigquery-cloud-sdk :datetime-diff]
+  [_driver _feat _db] true)
```

Comment:
this is in the bad examples here https://guide.clojure.style/#one-line-functions

```suggestion
(defmethod driver/database-supports? [:bigquery-cloud-sdk :datetime-diff]
  [_driver _feat _db]
  true)
```

Reply:
Updated in https://github.com/metabase/metabase/commit/aaa8ac665d597cc1f1d8e2485d3d59050e6c4b56